onebusaway-android/src/main/java/org/onebusaway/android/tripservice/TripService.java

```diff
@@ -137,7 +138,7 @@ public void cancelNotification(int id) {
 
         @Override
```

Comment:
@charles-b-stb Could you please completely remove `getNotification()` here?  You'll need to remove it from the interface as well.